SourceX/DiabloUI/diabloui.cpp

```diff
@@ -426,9 +426,9 @@ void LoadUiGFX()
 	LoadMaskedArt("ui_art\\focus42.pcx", &ArtFocus[FOCUS_BIG], 8);
 	LoadMaskedArt("ui_art\\cursor.pcx", &ArtCursor, 1, 0);
 #ifdef HELLFIRE
-	LoadArt("ui_art\\heros.pcx", &ArtHero, 6);
+	LoadArt("ui_art\\heros.pcx", &ArtHero, 6, NULL, 76);
 #else
-	LoadArt("ui_art\\heros.pcx", &ArtHero, 4);
+	LoadArt("ui_art\\heros.pcx", &ArtHero, 4, NULL, 76);

```

Comment:
No need to hardcode here, right?

Reply:
yeah, but what if someone creates Patch_rt.mpq that will modify heros.pcx for vanilla? :P Then we'll have the same issue :P

Reply:
OK, I'm not really sure how this is supposed to work
Wouldn't adding a hero require changes elsewhere in the code as well?

Reply:
yeah but having the frame height hardcoded isn't harmful in any way atm so why not do it? :P we know how many frames we want, we know their height, so there's no need to calculate their height by dividing total height by the number of frames, which might come out wrong like in my case